Dependencies/xr/Source/ARKit/XR.mm

```diff
@@ -124,6 +142,13 @@ @implementation SessionDelegate {
     return &updatedPlanes;
 }
 
+/**
+ Returns the set of all updated meshes since the last time we consumed mesh updates.
+ */
+- (std::set<ARMeshAnchor*, ARAnchorComparer>*) GetUpdatedMeshes  API_AVAILABLE(ios(13.4)){
```

Comment:
Everywhere you use `ARMeshAnchor` I think needs to be wrapped in a `#if (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_4)` check.